com.ca.lsp.cobol/lsp-core-cobol-parser/src/test/java/com/ca/lsp/core/cobol/visitor/LevenshteinDistanceTest.java

```diff
@@ -37,7 +37,7 @@
 
   @Test
   public void testDistance() {
-    errors.add(new SyntaxError(new Position("", 1, 1, 1, 1), null, "", 2));
+    errors.add(new SyntaxError(new Position("", 1, 1, 1, 1), null, "", 2, null));
```

Comment:
It looks like a good candidate for a factory method.

Reply:
I have factory method changes in another branch, don't want to have a merge conflict.